src/v2/ion/RemediationConstants.js

```diff
@@ -84,7 +84,8 @@ const AUTHENTICATOR_KEY = {
   GOOGLE_AUTHENTICATOR: 'google_authenticator',
   ON_PREM: 'onprem_mfa',
   RSA: 'rsa_token',
-  DUO: 'duo_native'
+  DUO: 'duo_native',
```

Comment:
You probably need to rebase the latest, @mauriciocastillosilva-okta have fixed the duo key I think

Reply:
This is based on master https://github.com/okta/okta-signin-widget/commit/514881e93ade56ba3740bb214a8b7767df21f997#diff-ada787b216ab157b8b42f52e05dc71006932f7e8e881de850cbfe217e8a4469fR86. 

His fix was made on the patch branch https://github.com/okta/okta-signin-widget/commit/54c84815b35c0ecbe3e389ac0bf2d03f503f4376. It will get fixed when its backported